### ISSUE TITLE
Tab buttons can overflow the window in the Video Settings screen 

### ISSUE THREAD

**Issue by seema84** [jellysquid3/Sodium#2976]
### Bug Description

From a GUI size greater than 2, the entire menu is no longer displayed. Some tabs are therefore not visible/accessible.

### Reproduction Steps

Version: Sodium 0.6.5 
1. 1080p resolution
2. GUI Size 3
3. add some Mods (Sodium Extras, LambDynamicLights etc.)

![Image](https://github.com/user-attachments/assets/c5a3561c-5b1b-4c75-981b-00b2590beb7b)

**Comment by douira** [comment]
This is simply what happens when other mods add their own tabs and we don't have a scrollable list for all the tabs. Fixed by #2837 when it gets merged. Until then, you can try using one of the mods that turn the sodium settings ui into one that includes a scrollable list of mods and categories.

**Comment by seema84** [comment]
oh sorry had not seen the PR
